### ISSUE TITLE
Is the challenge still active?

### ISSUE THREAD

**Issue by mariansam** [LiveOverflow/ctf#3]
You say, if we can find the flag. I tried almost everything. Is it still there? If yes, I will try looking for it once more. If not, I wasted my time.

**Comment by LiveOverflow** [comment]
yup, can still be solved. but don't waste your time. It's not that magical :D

**Comment by mariansam** [comment]
ok, i found it. Maybe not the way you want..

**Comment by NoahvdAa** [comment]
Found it!

**Comment by bence98** [comment]
Me too, although I waay over-engineered it at first :smile:

**Comment by breaker01** [comment]
found 😊

**Comment by Zchap1** [comment]
Cool

**Comment by yxmark** [comment]
been stuck for hours, can anyone give me a small hint?

**Comment by bence98** [comment]
This was a publicly accessible repository.

**Comment by yxmark** [comment]
> This was a publicly accessible repository.

Thx. Found it. Totally over engineered it..

**Comment by RaspberryPiFan** [comment]
LO{the_1nt3rnet_n3v3r_f0rgets}

**Comment by RaspberryPiFan** [comment]
SPOILER!!!!

**Comment by pitust** [comment]
Really good challange!

**Comment by mobeigi** [comment]
Found. Very common in CTF's with public web pages! 😄

**Comment by SmintGaming** [comment]
LO{the_1nt3rnet_n3v3r_f0rgets}

**Comment by mariansam** [comment]
@LiveOverflow thanks... 🤦

**Comment by machinexa2** [comment]
Does the flag has word dirty in it

**Comment by mobeigi** [comment]
@machinexa2 No it doesn't

**Comment by MrPancakes39** [comment]
Nice challenge, found
